index.md

```diff
@@ -26,9 +26,12 @@
 
 Vendor | Base Pricing | SSO Pricing | % Increase | Source | Date Added
 ------ | ------------ | ----------- | ---------- |------ | ----------
-Fictional Vendor 1 | $5 per u/m | $15 per u/m | 200% | vendor.org/pricing | 10/17/18
-Fictional Vendor 2 | $10 per u/m | $20 per u/m | 100% | vendor.org/pricing | 10/17/18
-Fictional Vendor 3 | $10 per u/m | $40 per u/m | 300% | vendor.org/pricing | 10/17/18
+DocuSign | $25 per u/m | $50 per u/m | 100% | [🔗](https://www.docusign.com/products-and-pricing) Quote | 2018-10-17
+Expensify | $5 per u/m | $9 per u/m | 80% | [🔗](https://www.expensify.com/pricing#features) | 2018-10-17
+Lucidchart | $7 per u/m | $CALL | ??? | [🔗](https://www.lucidchart.com/users/registerLevel) | 2017-10-17
```

Comment:
Is this accurate? Hard to tell from the pricing page where the SSO lives, and I don't see a $7 tier.